integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/DeleteExecution.scala

```diff
@@ -134,7 +134,7 @@ object DeleteExecution {
                        groupedRows.toIterator,
                        timestamp,
                        rowCountDetailsVO,
-                       carbonTable.isHivePartitionTable)
+                       segmentFile)
```

Comment:
Is it better to pass SegmentStatus instead of string, to have better encapsulation

Reply:
Ok, But passed Segment